src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -399,16 +433,16 @@ def set_scope(self, value):
     def handle_example(self, text, continue_flag):
         """ parses for the tutorial """
         cmd = text.partition(SELECT_SYMBOL['example'])[0].rstrip()
-        num = text.partition(SELECT_SYMBOL['example'])[2].strip()
+        selected_option = text.partition(SELECT_SYMBOL['example'])[2].strip()
```

Comment:
> text.partition(SELECT_SYMBOL['example'])[2].strip()

Could you please give some comments and example to explain this logic? Why the index of `text.partition(SELECT_SYMBOL['example'])` is `2`?

Reply:
Done